core/lib/lighthouse-compatibility.js

```diff
@@ -81,6 +81,22 @@ function upgradeLhrForCompatibility(lhr) {
         }
       }
 
+      // In 10.0, third-party-summary deprecated entity: LinkValue and switched to entity name string
```

Comment:
Hmm, what's the idea behind doing this? Rendering existing LHRs as links for this audit should be ok right?

Reply:
It would align the UI better towards 10.1. I see 10.0 as a point for breaking changes/expectations, so it might be odd to see links in the audit results in 10.0, and see just text 10.1. With this, we'd see no such functional regressions between the minor versions — with the "click-to-open-in-a-new-tab" coming in 10.1, which is an incremental change.

Reply:
I'd rather have less code here, this is really just for stuff that will _break_ the renderer.

Reply:
Sounds good. I'll remove the upgrade code.